src/cli.rs

```diff
@@ -81,6 +84,10 @@ impl Options {
                 .short("t")
                 .default_value(DEFAULT_TITLE)
                 .help("Defines the window title"))
+            .arg(Arg::with_name("class")
+                 .long("class")
+                 .default_value(DEFAULT_CLASS)
+                 .help("Defines WM_CLASS on supported platforms"))
```

Comment:
It might be nice change the description up a bit. Something like `Defines window class on X11`. This is probably closer to what a user would expect and should make it obvious where this is supported.

It might also be possible to make use of platform-specific code here. Since MacOS will never support the window class, it might be nice to skip this option for MacOS. However in my experience splitting Clap into multiple functions is kind of a mess, so it's probably okay the way it is right now.

Reply:
Skipping the option seems fine as long as we don't ever start using `--class` for something else on macOS. Which I can't imagine why we'd do.

Reply:
Yeah, but it's probably easier to just leave it in for now, like that every user will get the same stuff.